test/plugin_functional/test_suites/core_plugins/rendering.ts

```diff
@@ -53,9 +57,150 @@ export default function ({ getService }: PluginFunctionalProviderContext) {
       return window.__RENDERING_SESSION__;
     });
 
-  // Talked to @dover, he aggreed we can skip these tests that are unexpectedly flaky
-  describe.skip('rendering service', () => {
```

Comment:
😬
I had to unskip the suite itself, but I skipped the individual tests below because I am not sure if they are still flaky or not.